sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/RawNetwork.java

```diff
@@ -59,7 +59,7 @@ public String getLabelsCsv() {
 
     public List<String[]> getValidatedLabels() throws IllegalArgumentException {
         List<String[]> kvs = Arrays.stream(labelsCsv.split(","))
-                .map(s -> s.split(":"))
+                .map(s -> s.split(":", 2))
```

Comment:
Note: I added this to allow for a `:` as part of a network label's value, as that is sometimes needed.